### PR TITLE
docs: add apisix auth tech blog

### PR DESCRIPTION
**Pull request by SylviaBABY**
Changes:

add APISIX auth tech blog

### CODE REVIEW

File: website/blog/2021-09-07-how-to-use-apisix-auth.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+keywords: 
+- APISIX
+- 身份认证
+- 集中式
+description: 本文介绍了 Apache APISIX 的身份认证功能，从重要性和玩法使用上进行了详细介绍和细节使用
+tags: [technology]
+---
+
+> [@朱欣欣](https://github.com/starsz) 是[API7.AI](https://www.apiseven.com/zh)的系统工程师，也是 Apache APISIX 项目的 committer。开源爱好者，也是一个 Golang 开发者。现在他正在为 Apache APISIX 的 control plane 贡献代码并执行 code review。除此之外，他在 Apache APISIX 社区中非常活跃，乐于回答社区内的各种问题。
+>
```
Comment (yzeng25) [inline website/blog/2021-09-07-how-to-use-apisix-auth.md]: [@朱欣欣](https://github.com/starsz)，Apache APISIX 项目的 committer。

Comment (yzeng25) [reply website/blog/2021-09-07-how-to-use-apisix-auth.md]: 社区文章尽量不要出现公司title

Comment (SylviaBABY) [reply website/blog/2021-09-07-how-to-use-apisix-auth.md]: done

**Review by yzeng25 — APPROVED** [review]

**Comment by yzeng25** [comment]
Hi @liuxiran @bzp2010 , please approve when you are available, thanks!

File: website/blog/2021-09-07-how-to-use-apisix-auth.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
+---
+title: "使用 Apache APISIX 进行集中式身份认证及进阶玩法"
+author: "朱欣欣"
+authorURL: "https://github.com/starsz"
+authorImageURL: "https://avatars.githubusercontent.com/u/25628854?v=4"
+keywords: 
+- APISIX
+- 身份认证
+- 集中式
```
Comment (bzp2010) [inline website/blog/2021-09-07-how-to-use-apisix-auth.md]: Should "集中式" and "身份认证" be combined?

File: website/blog/2021-09-07-how-to-use-apisix-auth.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+![鉴权请求](https://static.apiseven.com/202108/1631005141578-f90cf948-4913-45cd-a28e-9e697ad197fe.png)
+
+通过 HTTP 库发起一个 HTTP 请求，我们可以利用第三方服务做一些认证、鉴权相关事情，然后解析返回结果。通过这种方式，我们可以做的事情就可以扩展很多。比如对接 Redis 或数据库，只要是通过 TCP 或 UDP 这种形式的，都可以通过 Serverless 插件进行尝试。
+
+### 配置上传
+
+完成自定义代码片之后，我们创建路由，并将代码片配置到 Serverless 插件。后面再通过访问 Apache APISIX 获取相应的信息反馈，测试插件是否生效。
+
+![配置上传](https://static.apiseven.com/202108/1631005184917-bc620c0b-d4c6-43f5-8450-4f5b2b9549e1.png)
+
```
Comment (liuxiran) [inline website/blog/2021-09-07-how-to-use-apisix-auth.md]: please remove extra lines

**Comment by liuxiran** [comment]
please fix the markdown lint errors, thanks

File: website/blog/2021-09-07-how-to-use-apisix-auth.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+创建消费者 foo。在消费者中，我们需要配置用户的认证信息，例如 `username` 为 foo，`password` 为 `bar`。
+
+![创建消费者](https://static.apiseven.com/202108/1631004937828-15ac5d8f-0e45-4c3d-94e8-2b180266b379.png)
+
+#### 步骤三：应用服务携带认证信息进行访问
+
+应用携带 `username:foo` ,`password: bar` 访问 Apache APISIX。Apache APISIX 通过认证，并将请求 Authorization 请求头携带至上游 `httpbin.org` 。由于 `httpbin.org` 中 get 接口会将请求信息返回，因此我们可以在其中观察到请求头 `Authorization`。
+
+![](https://static.apiseven.com/202108/1631004973305-4b209f79-f7de-41a2-994e-8877a6624d99.png)
```
Comment (liuxiran) [inline website/blog/2021-09-07-how-to-use-apisix-auth.md]: image name

File: website/blog/2021-09-07-how-to-use-apisix-auth.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+###  中级：进阶玩法
```
Comment (liuxiran) [inline website/blog/2021-09-07-how-to-use-apisix-auth.md]:
```suggestion
### 中级：进阶玩法
```

File: website/blog/2021-09-07-how-to-use-apisix-auth.md (hunk 4 above)
Comment (SylviaBABY) [reply website/blog/2021-09-07-how-to-use-apisix-auth.md]: done

File: website/blog/2021-09-07-how-to-use-apisix-auth.md (hunk 3 above)
Comment (SylviaBABY) [reply website/blog/2021-09-07-how-to-use-apisix-auth.md]: done

**Review by liuxiran — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**